src/AccountingApp.API/Data/AuthRepository.cs

```diff
@@ -64,9 +64,12 @@ private void CreatePasswordHash(string password, out byte[] passwordHash, out by
             }
         }
 
-        public Task<bool> UserExist(string username)
+        public async Task<bool> UserExist(string username)
         {
-            throw new System.NotImplementedException();
+            if(await _context.Users.AnyAsync(x=>x.Username == username))
```

Comment:
could be simplified by `return await  _context.Users.AnyAsync(x=>x.Username == username);`